_includes/templates/provisioning/coap-access-token-credentials-type.md

```diff
@@ -75,7 +75,7 @@ def collect_required_data():
     print("\n\n", "="*80, sep="")
     print(" "*10, "\033[1m\033[94mThingsBoard device provisioning with access token authorization example script. CoAP API\033[0m", sep="")
     print("="*80, "\n\n", sep="")
-    host = input("Please write your ThingsBoard \033[93mhost\033[0m or leave it blank to use default (thingsboard.cloud): ")
+    host = input("Please write your ThingsBoard \033[93mhost\033[0m or leave it blank to use default ({{hostName}}): ")
```

Comment:
Probably it will be better to use the same variable that will be set, in this case - coapHostName